test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -1154,52 +1155,53 @@ describe('CSOT spec prose tests', function () {
        *
        * 7. Verify that two `bulkWrite` commands were executed as part of the `MongoClient.bulkWrite` call.
        */
-      const failpoint: FailPoint = {
-        configureFailPoint: 'failCommand',
-        mode: {
-          times: 2
-        },
-        data: {
-          failCommands: ['bulkWrite'],
-          blockConnection: true,
-          blockTimeMS: 1010
-        }
-      };
 
-      let maxBsonObjectSize: number;
-      let maxMessageSizeBytes: number;
+      const writes: CommandStartedEvent[] = [];
 
       beforeEach(async function () {
         await internalClient
           .db('db')
           .collection('coll')
           .drop()
           .catch(() => null);
-        await internalClient.db('admin').command(failpoint);
-
-        const hello = await internalClient.db('admin').command({ hello: 1 });
-        maxBsonObjectSize = hello.maxBsonObjectSize;
-        maxMessageSizeBytes = hello.maxMessageSizeBytes;
+        await internalClient.db('admin').command(<FailPoint>{
+          configureFailPoint: 'failCommand',
+          mode: {
+            times: 2
+          },
+          data: {
+            failCommands: ['bulkWrite'],
+            blockConnection: true,
+            blockTimeMS: 1010
+          }
+        });
 
         client = this.configuration.newClient({ timeoutMS: 2000, monitorCommands: true });
+        client.on('commandStarted', filterForCommands('bulkWrite', writes));
       });
 
```

Comment:
This prose test incorporates changes from https://github.com/mongodb/specifications/pull/1672.

Reply:
ah in the `makeMultiBatchWrite` helper, got it